danger-kotlin/src/runnerMain/kotlin/com/danger/runner/cmd/Cmd.kt

```diff
@@ -0,0 +1,33 @@
+package com.danger.runner.cmd
+
+import platform.posix.system
+
+object Cmd : ICmd {
```

Comment:
Can this have `name` and `args` in the init? with a default for the args and use a different one for each command, then we don't have static state

Reply:
evaluating a more elegant solution with no state